app/src/main/java/at/bitfire/icsdroid/Constants.kt

```diff
@@ -14,4 +14,13 @@ object Constants {
 
     val USER_AGENT = "ICSx5/${BuildConfig.VERSION_NAME} (ical4j/${ical4jVersion} okhttp/${OkHttp.VERSION} Android/${Build.VERSION.RELEASE})"
 
+    /**
+     * Suggested user agents to allow compatibility with user agent blocking servers
+     */
+    val COMPATIBILITY_USER_AGENTS = mapOf(
+        "Chrome Android" to "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/134.0.0.0 Mobile Safari/537.3",
```

Comment:
Is this correct? The UA contains Mozilla and Gecko at the end. Also Mobile Safari